dev/io.openliberty.org.apache.myfaces.4.0_fat/publish/servers/faces40_facesContextGetLifecycleServer/server.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+    Copyright (c) 2022 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Server for testing the Faces 4.0 FacesContext.getLifecycle() API">
+
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+        <feature>faces-4.0</feature>
+        <feature>cdi-4.0</feature>
+    </featureManager>
```

Comment:
Should we create a new server for each test? Or could have a generic faces-cdi server?  Maybe something to follow up on later.

Reply:
I like having one server per test class, makes finding the logs etc easier. If we have some tests we want to group together we could.

Reply:
https://github.com/OpenLiberty/open-liberty/wiki/FAT-Tests also recommends this approach: `Use one test class per server `